tests/Adapters/Microsoft.Bot.Builder.Adapters.Slack.Tests/Files/InteractiveMessageBody.json

```diff
@@ -1,82 +1,80 @@
-{
-  "payload": {
-    "type": "interactive_message",
-    "actions": [
-      {
-        "name": "recommend",
-        "value": "recommend",
-        "type": "button"
-      }
-    ],
-    "callback_id": "comic_1234_xyz",
-    "team": {
-      "id": "T47563693",
-      "domain": "watermelonsugar"
-    },
-    "channel": {
-      "id": "C065W1189",
-      "name": "forgotten-works"
-    },
-    "user": {
-      "id": "U045VRZFT",
-      "name": "brautigan"
-    },
-    "action_ts": "1458170917.164398",
-    "message_ts": "1458170866.000004",
-    "attachment_id": "1",
-    "token": "testToken",
-    "original_message": {
-      "text": "New comic book alert!",
-      "attachments": [
-        {
-          "title": "The Further Adventures of Slackbot",
-          "fields": [
-            {
-              "title": "Volume",
-              "value": "1",
-              "short": true
-            },
-            {
-              "title": "Issue",
-              "value": "3",
-              "short": true
-            }
-          ],
-          "author_name": "Stanford S. Strickland",
-          "author_icon": "https://api.slack.comhttps://a.slack-edge.com/80588/img/api/homepage_custom_integrations-2x.png",
-          "image_url": "http://i.imgur.com/OJkaVOI.jpg?1"
-        },
-        {
-          "title": "Synopsis",
-          "text": "After @episod pushed exciting changes to a devious new branch back in Issue 1, Slackbot notifies @don about an unexpected deploy..."
-        },
-        {
-          "fallback": "Would you recommend it to customers?",
-          "title": "Would you recommend it to customers?",
-          "callback_id": "comic_1234_xyz",
-          "color": "#3AA3E3",
-          "attachment_type": "default",
-          "actions": [
-            {
-              "name": "recommend",
-              "text": "Recommend",
-              "type": "button",
-              "value": "recommend"
-            },
-            {
-              "name": "no",
-              "text": "No",
-              "type": "button",
-              "value": "bad"
-            }
-          ]
-        }
-      ]
-    },
-    "response_url": "https://hooks.slack.com/actions/T47563693/6204672533/x7ZLaiVMoECAW50Gw1ZYAXEM",
-    "trigger_id": "13345224609.738474920.8088930838d88f008e0",
-    "event": {
-      "thread_ts": "1568915625.001000"
-    }    
-  }
-}
\ No newline at end of file
+payload%22 %7B
```

Comment:
Is this format OK?
![image](https://user-images.githubusercontent.com/43762887/65799529-b7dc2c00-e14a-11e9-96d6-36b85f8004a7.png)

Reply:
yes. This format is a json made into url format. It should not have the spaces and look all "minified" but it's still passing the tests like that.